src/Illuminate/Collections/CaseMode.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace Illuminate\Support;
+
+enum CaseMode
+{
+    case UPPER;
```

Comment:
The few enums already added to Laravel's core use camel case for their cases.

- https://github.com/laravel/framework/blob/309c08791197af9c8b69611481908952522b9cd3/src/Illuminate/Console/View/TaskResult.php
- https://github.com/laravel/framework/blob/309c08791197af9c8b69611481908952522b9cd3/src/Illuminate/Database/Migrations/MigrationResult.php